sessionctx/variable/sysvar.go

```diff
@@ -349,14 +373,6 @@ func (sv *SysVar) checkInt64SystemVarWithError(value string) (string, error) {
 	return value, nil
 }
 
-// ValidateFromHook calls the anonymous function on the sysvar if it exists.
-func (sv *SysVar) ValidateFromHook(vars *SessionVars, normalizedValue string, originalValue string, scope ScopeFlag) (string, error) {
-	if sv.Validation != nil {
-		return sv.Validation(vars, normalizedValue, originalValue, scope)
-	}
-	return normalizedValue, nil
-}
-
```

Comment:
This function becomes obsolete because `Validate()` was introduced, replacing varsutil.go: ValidateSetSystemVar.